### DISCUSSION TITLE
I want to change my layout theme

### DISCUSSION THREAD

**Discussion by chenxiaofie** [PanJiaChen/vue-element-admin#4151] [Q&A] [answered]
Please help me how can i do change my layout them

**Comment by cocoguma** [comment, marked as answer]
If you want to switch between bright and dark, you can use App.setisdark (val) to switch themes.

**Comment by wusong4988398** [comment]
thanks you !
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PanJiaChen/vue-element-admin"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月17日(星期一) 下午5:02
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [PanJiaChen/vue-element-admin] I want to change my layout theme (Discussion #4151)





 
If you want to switch between bright and dark, you can use App.setisdark (val) to switch themes.
